src/main.js

```diff
@@ -1,10 +1,25 @@
 import Vue from 'vue'
 import App from './App'
```

Comment:
quibble, convention: first import libs from 3-rd party npm packages, only then import your files. The Only Correct Import order (tm) is: 1. built-in libraries 2. 3-rd party libs 3. internal project libs